container/CHANGELOG.md

```diff
@@ -0,0 +1,25 @@
+# Changelog
+
```

Comment:
```suggestion
[v1.2.0] (2024-07-25)
```

maybe we add this as well here

Reply:
Indeed 👍 . Removed it in the last commit by accident.